reservations/schema.py

```diff
@@ -116,18 +116,18 @@ class WinterStorageReservationInput(BaseReservationInput):
 
 class CreateBerthReservation(graphene.Mutation):
     class Arguments:
-        berth_reservation = BerthReservationInput(required=True)
+        berth_application = BerthApplicationInput(required=True)
```

Comment:
these changes still cause a change of the API. `test_reservations_graphql.py` should pass without any changes to the tests. you can add some TODOs to remind us to rename the objects later on though

Reply:
Would it make sense to presereve only the field names (`berth_reservation`) but already rename what we can (i.e. `BerthReservationInput -> BerthApplicationInput`)?

Or would it be better to wait and rename all the related fields at the same time?

Reply:
if you rename the input, it will change the API, the frontend will ask for regeneration of types. API should stay 100 % the same for now